test/_init_tests.js

```diff
@@ -363,6 +363,10 @@ function stubConsoleInfoLogWarn() {
     consoleInfoLogWarnSandbox.restore();
   }
   consoleInfoLogWarnSandbox = sinon.sandbox.create();
+  const {debugConsole} = window.__karma__.config;
+  if (debugConsole) {
+    return;
+  }
```

Comment:
Adding the logic here will result in `consoleInfoLogWarnSandbox` unnecessarily being created. You could move this to where `stubConsoleInfoLogWarn()` is called in `beforeEach()`.

https://github.com/ampproject/amphtml/blob/cbf3929bd9e867f38c18c1abeddd1957a7c46e29/test/_init_tests.js#L376-L384